### PR TITLE
Updated SPRING:Tandem and DLAB:SecVM

### PR DESCRIPTION
**Pull request by ineiti**

### CODE REVIEW

File: data/DLAB/projects.yaml [hunk 1]
```diff
@@ -31,11 +31,18 @@ projects:
...
         url: https://arxiv.org/pdf/1907.03373.pdf
+      - type: Paper
+        title: Privacy-Preserving Distributed Learning with Secret Gradient Descent
+        url: https://arxiv.org/pdf/1906.11993.pdf
       - type: Source code
         title: Client source code
         url: https://github.com/cliqz-oss/browser-core/tree/6945afff7be667ed74b0b7476195678262120baf/modules/secvm/sources
+      - type: Source code
+        title: Server source code
+        url: https://github.com/epfl-dlab/secvm-server
```
Comment (cgrigis) [inline data/DLAB/projects.yaml:42]: Isn't this is already listed in the `code` section above?

Comment (ineiti) [reply data/DLAB/projects.yaml:42]: Oups - correct. I replaced the `code` entry with the client code, as I cannot find the client code in the server code :) I'll ask this afternoon how to find it.

Comment (cgrigis) [reply data/DLAB/projects.yaml]: The client code is referenced in the README of the server code, which I think is why I had put that one in the main `code` section...

Comment (ineiti) [reply data/DLAB/projects.yaml]: I need holidays... So I only added the 'updated' paper here.

**Review by cgrigis — APPROVED** [review]

**Pull request merged**